database/config.js

```diff
@@ -1,6 +0,0 @@
-module.exports = {
-  host: 'localhost',
-  user: 'root',
-  password: '',
-  database: 'tier'
```

Comment:
Maybe consider trailing comma? The styleguide like sit and there are diff benefits when you push